ambari-web/app/messages.js

```diff
@@ -3198,7 +3196,7 @@ Em.I18n.translations = {
   'tableView.filters.all': 'All',
   'tableView.filters.filtered': 'Filtered',
   'tableView.filters.clearFilters': 'Clear filters',
-  'tableView.filters.itemsPerPage': 'Items per page:',
+  'tableView.filters.rowsPerPage': 'Rows per page:',
```

Comment:
Not a fan of "Rows per page" as it is a bit too literal. "Items" wasn't much better. I would rather see this string be contextual, like "Hosts per page" or whatever the table rows represent.

Reply:
Currently the way this is implemented is using App.PaginationView which doesn't allow this customization, so this has to be generic (there could be tables not having any hosts but processes or services or components instead). We can of course change the way view is implemented to make it more customizable but that requires a larger effort and the change would need to be reflected in other parts of the code. We can leave it for now, and can re-visit it later if there is a more pressing need.